Linear Regression/Linear_Regression.py

```diff
@@ -0,0 +1,78 @@
+"""
+Linear regression
+Arguments - alpha (by default .001)
+            epochs(by default 100)
+            plot : bool - plot the cost vs epoch graph  
+function -  fit : (x, y) - train the data
+            predict : (x) - predict the new y using previus trained model
+            this module also normalize data brfore training that increase training efficiency
+"""
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+
+class LinearRegression:
+    
+    def __init__(self):
```

Comment:
You have docstring issue in here. Missing docstring in __init__

Reply:
Yes, I will fix that